rabbitmq/connection_integration_test.go

```diff
@@ -0,0 +1,171 @@
+package rabbitmq
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	amqp "github.com/rabbitmq/amqp091-go"
+	"github.com/stretchr/testify/require"
+	tc "github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func startRabbitMQContainer(t *testing.T) (tc.Container, string) {
+	t.Helper()
+
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+
+	username := "guest"
+	password := "guest"
+
+	req := tc.ContainerRequest{
+		Image:        "rabbitmq:3.13-management",
```

Comment:
Is this the version used in production? I used the latest `4-management` version in the cloudery dev and test config.